wai-aria/heading/invalid-nesting.tentative.html

```diff
@@ -0,0 +1,176 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>WAI-ARIA tests for invalidly nested elements with heading role</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/resources/testdriver-actions.js"></script>
+  <script src="/wai-aria/scripts/aria-utils.js"></script>
+</head>
+
+<body>
+
+  <h1>WAI-ARIA tests for invalidly nested heading elements</h1>
+  <p>Equivalent ARIA patterns tested in <code>/html-aam/heading/invalid-nesting.html</code>.</p>
+
```

Comment:
Since those all pass, and several of these fail (with identical results in the engines), there are a few requests and options... If possible, cross reference the spec text (or texts from multiple specs) that conflicts with the the current browser consensus.

Since it seems like the tentative file may take some time to discuss with the WG and browser teams, you might consider moving all the passing tests in main file, and leave only the failing tests in a tentative file (perhaps in a different PR)

Reply:
Even if we do learn that the spec says the browser is wrong, engine consensus is a powerful motivator to question whether the spec is indeed correct. It may be, but we should not forego implementation consensus lightly.